typings/index.d.ts

```diff
@@ -59,7 +59,7 @@ import {
 
 //#region Classes
 
-export class Activity {
+export class Activity extends Base {
```

Comment:
Activity does not seem to be extending Base.

Reply:
Ah, I meant to do the Application class